navigation/libandroid-navigation/src/main/res/layout/layout_notification_default.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can these attributes be pushed to the `LinearLayout` below (and then remove this outermost parent)?  The child is one `LinearLayout`